crates/solidity/outputs/cargo/crate/src/ir/_builder.rs.jinja2

```diff
@@ -1,6 +1,5 @@
-{%- if not builder -%}
-  // Not rendering builder for Intermediate Language
-{%- else -%}
+{% macro render_builder(language) -%}
```

Comment:
We are using both `ir` and `language` in the codebase to refer to the same concept. WDYT of consolidating to use `language` everywhere, and moving them to the `backend` module?

Putting the common templates in:
- `crates/solidity/outputs/cargo/crate/src/backend/languages/common/_builder.rs.jinja2`
- ...

And the individual language instances next to it:
- `crates/solidity/outputs/cargo/crate/src/backend/languages/l1_structured_ast/builder.rs.jinja2`
- ...

Reply:
That's a good point. In the generation code we're using `IrModel` and generally `ir`  seems to be the more common term used to name these trees, so maybe we should stick with that and organize it as:
- `crates/solidity/outputs/cargo/crate/src/backend/ir/common/_builder.rs.jinja2`
- `crates/solidity/outputs/cargo/crate/src/backend/ir/l1_structured_ast/builder.rs.jinja2`

I also realized that we're using `passes` to named the IR language transformations, but also the backend/binder passes. It's a very minor thing but can be confusing.

Another thing I'm thinking is that we should probably try get rid of `l1_structured_ast` altogether and generate `l2` directly from the CST. But that's definitely out of scope for this PR.